code_challenges/advent_of_code/chess_game.py

```diff
@@ -0,0 +1,30 @@
+import hashlib
+
+
+def find_pass_code(code):
+    used_hashes = []
+    number = 0
+    password = ""
+    while len(used_hashes) < 8:
```

Comment:
is this while loop correct?  You never need more than 7?

Reply:
doesnt this always give me 8? because when the len(used_hashes) == 7 it enters the block and then adds to used_hashes and then the len is 8 and it doesnt enter

Reply:
Sure, I was off by one, the question was meant to ask if you need a fixed number. I you only ever need 8 okay.